modules/phase_field/src/action/GrandPotentialKernelAction.C

```diff
@@ -118,6 +120,9 @@ GrandPotentialKernelAction::act()
                "switching_function_names");
   if (M.size() != n_w)
     mooseError("M and chemical_potentials should be vectors of the same length.");
+  if (aniso.size() != n_w)
+    mooseError(
```

Comment:
I suggest `paramError("anisotropic", "Provide as many values as entries in 'chemical_potentials'.");`

Reply:
```suggestion
    paramError("anisotropic", "Provide as many values as entries in 'chemical_potentials'.");
```